src/components/surveyQuestions/SurveyQuestion.js

```diff
@@ -9,24 +9,24 @@ import {
   SurveyOptionsSection,
 } from './styles/surveyQuestionStyles';
 
-const SurveyQuestion = (props) => {
+const SurveyQuestion = ({item, formState, updateState}) => {
```

Comment:
I like what you did here. It makes it more clear where the information is coming from.